### PR TITLE
Adds hand-drawn image for circuit unitary factorization to Haar demo

### PR DESCRIPTION
**Pull request by josh146**
- Adds hand-drawn image for circuit unitary factorization to Haar demo
- Updates the spacing around images
- Adds some kerning to the integrals

### CODE REVIEW

File: demonstrations/tutorial_haar_measure.py [hunk 1]
```diff
@@ -393,11 +392,13 @@ def haar_random_unitary():
 # :math:`SU(N)` operation can be implemented by sandwiching an :math:`SU(2)`
 # transformation between two :math:`SU(N-1)` transformations, like so:
 #
+#  |
```
Comment (josh146) [inline demonstrations/tutorial_haar_measure.py:395]:
```suggestion
# |
```

File: demonstrations/tutorial_haar_measure.py [hunk 2]
```diff
@@ -59,7 +59,7 @@
 
 .. math::
 
-    V = \int_0^{r} \int_0^{2\pi} \int_0^{\pi} d\rho d\phi d\theta = 2\pi^2 r
+    V = \int_0^{r} \int_0^{2\pi} \int_0^{\pi} d\rho~ d\phi~ d\theta = 2\pi^2 r
```
Comment (glassnotes) [inline demonstrations/tutorial_haar_measure.py:62]: I didn't know you could do this! Looks much better :100:

File: demonstrations/tutorial_haar_measure.py [hunk 3]
```diff
@@ -370,10 +370,9 @@ def haar_random_unitary():
 # ways to decompose the unitary. Such decompositions rewrite elements in
 # :math:`SU(N)` acting on :math:`N` modes as a sequence of operations acting
 # only on 2 modes, :math:`SU(2)`, and single-mode phase shifts.  Shown below are
-# three examples [#deGuise2018]_, [#Clements2016]_, [#Reck1994]_ (images taken
-# from [#deGuise2018]_):
+# three examples [#deGuise2018]_, [#Clements2016]_, [#Reck1994]_:
 #
-# .. figure:: /demonstrations/haar_measure/sun_decomp.svg
+# .. figure:: /demonstrations/haar_measure/unitaries.png
```
Comment (glassnotes) [inline demonstrations/tutorial_haar_measure.py:375]: Nice!! :star_struck:

**Review by glassnotes — APPROVED** [review]
Thanks @josh146, greatly appreciated!

**Pull request merged**